schema/testdata/test_data.go

```diff
@@ -0,0 +1,13 @@
+package testdata
+
+func GetTestStructWithPrivateFieldPresent() TestStructWithPrivateField {
+	return TestStructWithPrivateField{
+		Field1: "test",
+		field2: "private",
+	}
+}
```

Comment:
Is this even referenced? I only see a direct reference to `testdata.TestStructWithPrivateField`...

Reply:
That's OK, if the separation makes sense. Adding test "infrastructure" that's not actually used yet is a bit weirder than doing the equivalent for a real interface; but as long as it wasn't left here by accident, that's fine.